pkg/store/consul/consultest/fake_preparer.go

```diff
@@ -51,11 +51,7 @@ func (f *FakePreparer) Enable() {
 					f.logger.Errorf("Error getting all pods: %v", err)
 					continue
 				}
-				// Make a copy to know what to delete
-				podsToDelete := make(map[consul.ManifestResult]struct{})
-				for _, manifestResult := range allPods {
```

Comment:
this could have still been valid, but it would have needed to range over Reality, not Intent.